### ISSUE TITLE
How to explicitly define response for a route

### ISSUE THREAD

**Issue by amsal** [danielgtaylor/huma#246]
I want to return a binary image response from one of my routes. 

https://go.dev/play/p/YlsBFq25iaK

But the generated openapi schema has few definitions I do not want. 

```json
"paths": {
    "/image": {
      "post": {
        "responses": {
          "200": {
            "content": {
              "application/json": {
                "schema": {
                  "contentEncoding": "base64",
                  "type": "string"
                }
              },
              "image/png": {
                "schema": {
                  "format": "binary",
                  "type": "string"
                }
              }
            },
            "description": "Image response",
            "headers": {
              "Content-Type": {
                "schema": {
                  "type": "string"
                }
              }
            }
          },
          "default": {
            "content": {
              "application/problem+json": {
                "schema": {
                  "$ref": "#/components/schemas/ErrorModel"
                }
              }
            },
            "description": "Error"
          }
        }
      }
    }
  }
```

Is it possible to change the success response to this? (remove the json response and content-type header)

```json
  "200": {
    "content": {
      "image/png": {
        "schema": {
          "format": "binary",
          "type": "string"
        }
      }
    },
    "description": "Image response"
  }
```

**Comment by danielgtaylor** [comment]
@amsal the fact that `application/json` is present appears to be a bug, thanks for reporting! I'll fix this ASAP.

As for the `Content-Type` header, Huma will document every header in your response and that is by design. I recommend keeping it, but if you absolutely want to remove it you can do so:

```go
delete(api.OpenAPI().Paths["/image"].Post.Responses["200"].Headers, "Content-Type")
```

https://go.dev/play/p/8OR1Ft4BYAY

**Comment by danielgtaylor** [comment]
@amsal the extra `application/json` response should be fixed in #247, and I'll publish a new release with the fix soon.
